src/routes/index.ts

```diff
@@ -21,7 +21,6 @@ export default (): express.Router => {
   router.patch('/:id', updateUser);
   router.delete('/:id', deleteUser);
 
```

Comment:
This doesn't belong here. I will not merge a pr with this index. Please reference how other index files are. 
```suggestion
```

Reply:
> This doesn't belong here. I will not merge a pr with this index. Please reference how other index files are.

can you recommend me what todo? this is how it's writen on develop and I didn't even work on these routes

Reply:
> This doesn't belong here. I will not merge a pr with this index. Please reference how other index files are.


this is how index files is structured on develop branch.. what should I do??

Reply:
> > This doesn't belong here. I will not merge a pr with this index. Please reference how other index files are.
> 
> this is how index files is structured on develop branch.. what should I do??

![develop-structure](https://github.com/atlp-rwanda/e-commerce-serge-30-bn/assets/77304295/abeabe9f-3021-4c92-b86c-6739924aa440)

Reply:
This is a team. Just because you didn't work on the file doesn't mean you don't correct it. I've told you to check how other index.ts files are. Did you do that and didn't know what to do? @Ke-bean

Reply:
> This is a team. Just because you didn't work on the file doesn't mean you don't correct it. I've told you to check how other index.ts files are. Did you do that and didn't know what to do? @Ke-bean

was confused before but now got an idea and working on it. will push ASAP